src/TraffiLearn.WebAPI/Controllers/UsersController.cs

```diff
@@ -85,9 +195,29 @@ public async Task<IActionResult> GetLoggedInUserComments()
         #region Commands
 
 
+        /// <summary>
+        /// Downgrades the account of a user with a specific ID.
+        /// </summary>
+        /// <remarks>
```

Comment:
Add some more explanation of how it works (if an account was with a role 'Admin', it becomes 'RegularUser', 'RegularUser' cannot be downgraded etc.)